apis/pkg/v1alpha1/package_types.go

```diff
@@ -44,6 +44,12 @@ type PackageSpec struct {
 	// Default is IfNotPresent.
 	// +optional
 	PackagePullPolicy *corev1.PullPolicy `json:"packagePullPolicy,omitempty"`
+
+	// IgnoreCrossplaneConstraints indicates to the package manager whether to
+	// honor Crossplane version constrains specified by the package.
+	// Default is false.
+	// +optional
+	IgnoreCrossplaneConstraints *bool `json:"ignoreCrossplaneConstraints,omitempty"`
```

Comment:
@hasheddan Given that it's assumed to be a rare case, would it make sense to make this an annotation instead of a first class spec field?

Reply:
I am pretty indifferent. The fact that it will be present for all package types kind of makes me lean towards making it a field, but either way could be fine

Reply:
@negz do you have any opinion on this? I am kind of leaning towards it being a dedicated spec field because it is more discoverable, but I don't feel _super_ strongly.

Reply:
Yeah, I'd lean slightly toward the optional field here too.